ACE/ace/OS_NS_unistd.inl

```diff
@@ -1178,10 +1183,12 @@ ACE_OS::unlink (const char *path)
 # elif defined (ACE_LACKS_UNLINK)
   ACE_UNUSED_ARG (path);
   ACE_NOTSUP_RETURN (-1);
-# elif defined (ACE_UNLINK_EQUIVALENT)
-  ACE_OSCALL_RETURN (ACE_UNLINK_EQUIVALENT (path), int, -1);
 # else
+#if defined (_MSC_VER) && (_MSC_VER >= 1800)
+  ACE_OSCALL_RETURN (::_unlink (path), int, -1);
```

Comment:
We have now support for ACE_UNLINK_EQUIVALENT which you removed, this has similar functionality. See also all other ACE_*_EQUIVALENT changes you removed, please correct this in your merge